python/libcuopt/libcuopt/_grpc_server_wrapper.py

```
@@ -0,0 +1,16 @@
+import subprocess
+import sys
+
+
+def main():
+    """
+    This connects to the gRPC server binary situated under libcuopt/bin folder.
+    """
+    server_path = os.path.join(
+        os.path.dirname(__file__), "bin", "cuopt_grpc_server"
+    )
+    sys.exit(subprocess.call([server_path] + sys.argv[1:]))
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)